README-ENGINES.md

```diff
@@ -185,7 +185,7 @@ the shared-library ENGINE implementation. If this command succeeds, the
 (copy of the) 'dynamic' ENGINE will magically morph into the ENGINE
 that has been loaded from the shared-library. As such, any control
 commands supported by the loaded ENGINE could then be executed as per
-normal. Eg. if ENGINE "foo" is implemented in the shared-library
```

Comment:
Would it be better to replace with "For example," or "For instance,"? "e.g." is typically used in parenthesized expressions (though not exclusively), so it seems odd here.

Reply:
Replaced